lib/fastlane/plugin/wpmreleasetoolkit/models/file_reference.rb

```diff
@@ -10,7 +10,7 @@ def initialize(params = {})
       end
   
       def source_contents
-        return File.read(secrets_repository_file_path) unless self.encrypt
+        return File.read(secrets_repository_file_path) unless (self.encrypt || ENV.key?('CIRCLECI'))
```

Comment:
Since this doesn't seem specific to CircleCI, but any CI provider, what do you think about using `CI` instead? It's one of the [CircleCI built-in env vars](https://circleci.com/docs/2.0/env-vars/#built-in-environment-variables), and is supported by [GitHub](https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables#default-environment-variables), [Travis CI](https://docs.travis-ci.com/user/environment-variables/#default-environment-variables), and [Buildkite](https://buildkite.com/docs/pipelines/environment-variables) too.

```suggestion
        return File.read(secrets_repository_file_path) unless (self.encrypt || ENV.key?('CI'))
```

Reply:
I chose `CIRCLECI` because it's the one used by Fastlane::is:ci ([#](https://github.com/fastlane/fastlane/blob/ce06c2156628637dbd74f43f87e5bf37c1a42111/fastlane_core/lib/fastlane_core/helper.rb#L73)) which we use in other places so that we have a coherent behaviour. 
I actually tried to use that function here, but it doesn't work (it always returns false). 
I dug a bit into the Fastlane code and it looks like the issue is related to some very in depth mechanisms of Fastlane. I'm adding a note to the code to revisit this with future versions of Fastlane.

Reply:
I'll also open a PR for this in Fastlane.